keps/sig-scheduling/20190312-pod-affinity-and-anti-affinity-support-gt-and-lt-operators.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Pod affinity/anti-affinity supports Gt and Lt operators
+authors:
+  - "@wgliang"
+owning-sig: sig-scheduling
+reviewers:
+  - "@bsalamat"
+  - "@k82cn"
+  - "@Huang-Wei"
+approvers:
+  - "@bsalamat"
+  - "@k82cn"
+creation-date: 2019-02-22
+last-updated: 2019-03-12
+status: provisional
+---
+
+ # Pod affinity/anti-affinity supports Gt and Lt operators
+
+ ## Table of Contents
+
+* [Summary](#summary)
+* [Motivation](#motivation)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+  * [User Stories](#user-stories)
+  * [Risks and Mitigations](#risks-and-mitigations)
+* [Design Details](#design-details)
+  * [Content](#content)
+  * [Test Plan](#test-plan)
+  * [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+ ## Summary
+
+ Extend the `Pod` affinity/anti-affinity operators to support `Gt` and `Lt` to provide 
+ users with more elegant Pod label selection capabilities.
+
+ ## Motivation
+
+  We know that `Node` affinity/anti-affinity currently supports `In`, `NotIn`, `Exists`, 
+`DoesNotExist`, `Gt`, `Lt`. But Pod affinity/anti-affinity only works with regular 
+label selectors: `In`, `NotIn`, `Exists`, `DoesNotExist`.
+
+ This is not an ideal situation if users want to put pods based on the label range. 
+ `Pod` affinity/anti-affinity support for `Gt` and `Lt` operators will give users more 
+ control.
+
+ ### Goals
+
+- `Pod` affinity/anti-affinity support for `Gt` and `Lt` operators.
+- `Gt` and `Lt` will have the same status and influence as the original operators (`In`, 
+`NotIn`, `Exists`, `DoesNotExist`).
+- `Gt` and `Lt` will work with `requiredDuringSchedulingIgnoredDuringExecution`(predicate, hard requirements) and `preferredDuringScheduling`(priority, soft requirements).
+
+ ### Non-Goals
+
+- Just add `Gt` and `Lt` operator functions without changing their original definition.
+- Not changing the behavior of other label selectors, such as `ReplicaSets`, `Daemonsets`, etc.
```

Comment:
...I think you have an extra "not" here, unless this is saying that it is a goal to change those things?

Reply:
Yes, you are right. Thank you for correcting me. I've updated it.